src/SpotMarketOracle.sol

```diff
@@ -31,10 +31,14 @@ contract SpotMarketOracle is IExternalNode {
         (, uint128 marketId) = abi.decode(parameters, (address, uint128));
 
         uint256 synthAmount;
+        bool useStrictStalenessTolerance;
         for (uint256 i = 0; i < runtimeKeys.length; i++) {
             if (runtimeKeys[i] == "size") {
                 synthAmount = uint256(runtimeValues[i]);
-                break;
+            }
+
+            if (runtimeKeys[i] == "useStrictStalenessTolerance") {
+                useStrictStalenessTolerance = _bytes32ToBool(runtimeValues[i]);
```

Comment:
here too, if key is not found, the default value will be false: to not use strict staleness tolerance